server/bleep/src/periodic/remotes.rs

```diff
@@ -1,7 +1,7 @@
 use std::{
```

Comment:
Maybe worth using `tokio::time::timeout` rather than the `select!` block on line 313 of this file?